ext/tags.go

```diff
@@ -19,30 +19,32 @@ var (
 	//////////////////////////////////////////////////////////////////////
 
 	// SpanKind hints at relationship between spans, e.g. client/server
-	SpanKind = spanKindTag("span.kind")
+	SpanKind = spanKindTagName("span.kind")
 
 	// SpanKindRPCClient marks a span representing the client-side of an RPC
 	// or other remote call
-	SpanKindRPCClient = SpanKindEnum("client")
+	SpanKindRPCClientEnum = SpanKindEnum("client")
```

Comment:
Sorry I should wait longer before merging changes. I'll be happy to apply the fixes you recommend. I'm sympathetic to your statement, and I would prefer if we called what are currently "tag names" instead "labels".

I'm not sure what you mean about conflating tags and options, though there is already some confusion over Tags (a field in StartSpanOptions, also a map[string]interface{} type).  My understanding is that OT uses "Tag" to refer to a pair of key and value.

To the specific comment here, about consistency, I'm not sure. There are only two SpanKind*Enum values here, and they are different types than the other names (Component, ...--tag names or what I would call labels) so I don't see a need for consistency. I'm not sure we need them, since SpanKindRPC(Server|Client).Value is equivalent.